.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file in a subsequent pull request and add the .vscode folder to your .gitignore file.